.gitignore

```diff
@@ -0,0 +1,2 @@
+__pycache__
```

Comment:
you must delete those folders because they are currently in the repo

Reply:
I did the .gitignore and put the name of the folders right and I don't know why they keep coming up